journey/pepr-build-wasm.ts

```diff
@@ -9,17 +9,24 @@ import { resolve } from "path";
 
 import { cwd } from "./entrypoint.test";
 
+// test npx pepr build -o dst
+const outputDir = "dst";
 export function peprBuild() {
+
+  it("should build artifacts in the dst folder", async () => {
+    execSync(`mkdir ${outputDir}`, { cwd, stdio: "inherit" });
```

Comment:
I'm being a little pedantic here, but... I think I'd prefer that we used node methods to interact with the underlying system where we can... just so we get/stay in the habit of not assuming the programs/versions/supported-flags that are installed on dev machines (and have a better time handling errors, etc.). :thinking: 

Would you be okay with moving to something like this..?
```suggestion
    await fs.mkdir(outputDir, { recursive: true })
```
Note: added that `recursive` option so that if anyone tries to add more path parts to the `outputDir` variable, node will just create the nested structures -- a bit like adding `-p / --parents` to mkdir would.

Reply:
100% good eye, should have thought of thought